workspaces/ui/src/contexts/RfcContext.js

```diff
@@ -93,7 +86,16 @@ function BaseRfcStore(props) {
         const eventStore = Facade.makeEventStore();
         global.eventStore = eventStore;
         if (initialEventsString) {
+          const markerNameStart = 'rfcContextAddEventsStart';
+          const markerNameStop = 'rfcContextAddEventsStop';
+          //performance.mark(markerNameStart);
```

Comment:
I did a little profiling and this is usually taking about 30-50ms compared to about 200ms for DiffManager.updateInteractions. Once I implement timing and add some larger scenarios to our snapshot tests we should get a better sense of any bottlenecks within the core.

Reply:
Did you end up saving any profiles to share? Would be interesting to see how it compares between machines / deployments.